subgen.py

```diff
@@ -50,7 +50,8 @@ def convert_to_bool(in_bool):
 procmediaonplay = convert_to_bool(os.getenv('PROCMEDIAONPLAY', True))
 namesublang = os.getenv('NAMESUBLANG', '')
 webhookport = int(os.getenv('WEBHOOKPORT', 9000))
-word_level_highlight = convert_to_bool(os.getenv('WORD_LEVEL_HIGHLIGHT', False))
+word_level_highlight = convert_to_bool(os.getenv('WORD_LEVEL_HIGHLIGHT', True))
```

Comment:
This needs to stay default False, otherwise lots of people will complain when their subs start highlighting words as they are displayed.